examples/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "basic-google-map",
+  "version": "1.0.0",
+  "private": true,
+  "description": "A sample Google Map setup with the MapProvider component.",
+  "main": "index.js",
+  "scripts": {
+    "clean-examples": "rm -rf public .parcel-cache",
+    "start:map": "PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234",
+    "start:map-markers": "PARCEL_AUTOINSTALL=false parcel serve ./google-map-with-markers/index.html --dist-dir public --port 1234"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@ubilabs/google-maps-react-hooks": "^1.5.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Comment:
update to react 18?

Reply:
Can only update this when the new dependency update is released. Otherwise it is conflicting when starting the examples because it is using `"@ubilabs/google-maps-react-hooks": "^1.5.1"`. Will need to wait for the release.